packages/limit-orders-lib/src/utils/queries/stoplimit.ts

```diff
@@ -140,20 +141,22 @@ export const queryPastOrders = async (
   }
 };
 
+const checkExpiration = (allOrders: StopLimitOrder[]): StopLimitOrder[] =>
+  allOrders.map((order: StopLimitOrder) => {
+    order.isExpired =
+      Date.now() > (parseInt(order.createdAt) + MAX_LIFETIME) * 1000;
+    return { ...order };
+  });
+
 export const _getUniqueOrdersWithHandler = (
   allOrders: StopLimitOrder[]
 ): StopLimitOrder[] =>
-  [...new Map(allOrders.map((order) => [order.id, order])).values()]
+  [
+    ...new Map(
+      checkExpiration(allOrders)
+        // filter out wrong module
+        .map((order) => [order.id, order])
+    ).values(),
```

Comment:
replace with _getUniqueOrdersWithExpiry

First get array of unique orders

Then addExpiry to them and return